e2e/testsuite/testsuite.go

```diff
@@ -224,6 +227,13 @@ func (s *E2ETestSuite) GetChains(chainOpts ...testconfig.ChainOptionConfiguratio
 // Once the broadcast response is returned, we wait for a few blocks to be created on both chain A and chain B.
 func (s *E2ETestSuite) BroadcastMessages(ctx context.Context, chain *cosmos.CosmosChain, user *ibc.Wallet, msgs ...sdk.Msg) (sdk.TxResponse, error) {
 	broadcaster := cosmos.NewBroadcaster(s.T(), chain)
+
+	configureGasFactoryOpt := func(factory tx.Factory) tx.Factory {
+		return factory.WithGas(500000)
+	}
+
+	broadcaster.ConfigureFactoryOptions(configureGasFactoryOpt)
```

Comment:
This was required to successfully exec group proposals

Reply:
should we make the gas argument a global const?

Reply:
Done